app/models/user.rb

```diff
@@ -179,6 +180,10 @@ def resources
     user_permissions.map(&:resource)
   end
 
+  def super_admin?
```

Comment:
Just a helper on User to check if they are `super_admin?` This allows us to guard-clause/by-pass our way through any access check anywhere.